docs/visual-testing/mobile-native-testing.md

```diff
@@ -25,6 +25,12 @@ When writing a visual test for mobile apps, we recommend the following:
 ## Limitations
 
 The following features are not yet available for mobile app testing:
-- Full page screenshots
 - DOM capture and inspection
 - [Selective diffing](./selective-diffing.md)
+
+Native full page screenshots are in beta and may have bugs.
```

Comment:
Native full-page screenshots are currently in beta and may have unexpected behavior.